pages/api/feedback/index.js

```diff
@@ -0,0 +1,68 @@
+import { getSession } from 'next-auth/client'; // Session handling
+import { getPerformances } from '../performances/collect';
+import { getSchools } from '../schools/collect';
+import { transporter } from 'aws/index';
+
+export default async (req, res) => {
+  const session = await getSession({ req });
+
+  if (session) {
+    const { schoolIDs } = req.query;
+
+    // Set filter to obtain schools with schoolIDs passed in. If schoolIDs is not passed in, we want to get all schools.
+    const schoolFilter = schoolIDs ? { id: { in: schoolIDs.split(',').map(i => +i) } } : {};
+    const schools = await getSchools(schoolFilter);
+
+    // If no schools are returned from the filter
+    if (!schools.length) return res.status(400).json({ error: 'No schools to email feedback.' });
+
+    // Filter for obtaining performances for the schools
+    const performanceFilter = {
+      school_id: {
+        in: schools.map(school => school.id),
+      },
+    };
+
+    // Obtain all performances matching the performanceFilter
+    const performances = await getPerformances(performanceFilter);
+
+    // Build a map where we can access all the performances of a school using the school id
+    let performancesMap = {};
```

Comment:
```suggestion
    const performancesMap = {};
```

Not reassigning, so `const` should be fine.